introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, my name is Greg and I am taking the Mastering GitHub course at codeschool.com.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?